src/CUITeTest/SearchConfigurations/BaseConfiguratorTest.cs

```diff
@@ -0,0 +1,63 @@
+﻿using CUITe.SearchConfigurations;
+using Microsoft.VisualStudio.TestTools.UITesting;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace CUITeTest.SearchConfigurations
+{
+    using System;
```

Comment:
We have the resharper settings file. Is it a matter of it being configured wrong?